llmware/library.py

```diff
@@ -664,7 +993,27 @@ def delete_library(self, library_name=None, confirm_delete=False):
     def update_block (self, doc_id, block_id, key, new_value):
 
         """ Convenience method to update the record of a specific block - identified by doc_ID and block_ID
```

Comment:
Delete space before you start the doc.
```suggestion
        """Convenience method to update the record of a specific block - identified by doc_ID and block_ID
```